apps/organisations/templates/a4_candy_organisations/organisation_form.html

```diff
@@ -1,12 +1,12 @@
-{% extends "a4dashboard/base_dashboard.html" %}
+{% extends "a4dashboard/organisation_form_base.html" %}
 {% load i18n %}
 
 {% block title %}{% trans "Settings" %} &mdash; {{ block.super }}{% endblock %}
 
-{% block dashboard_content %}
+{% block organisation_form %}
 <div class="container">
     <div class="row">
-        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
+        <div class="col-md-10 col-lg-10 offset-md-1">
```

Comment:
@phillimorland I am not so sure what I did here :) Before the form was really narrow and it looked better like this, but maybe you can check again if this makes sense?

Reply:
@Rineee nice! the class names dictate the amount of columns the content of the div will take up at each size of screen so now this will be 10 columns wide on both medium and large and the offset centers the content. As bootstrap uses the last class name for all bigger screens we don't need the extra col-lg-10 so i'll delete it :)

Reply:
Ok cool, thank you for the explanation!!